### PR TITLE
Add instructions on project

### PR DESCRIPTION
**Pull request by Montsant19**

### CODE REVIEW

File: packages/vscode-extension/README.md [hunk 1]
```diff
@@ -30,16 +30,24 @@ See more details about installation in <a href="https://system.netsuite.com/app/
...
 ### How to Trigger the Command Palette
 Interact through the Command Palette to use the SuiteCloud Extension for Visual Studio Code. To trigger it, click **View** > **Command Palette** or use the relevant shortcut:
 * Ctrl+Shift+P if you are using Windows or Linux.
 * Cmd+Shift+P if you are using Mac OS. 
 
 <p align="center"><a href="#"><img src="resources/CommandPalette.png" alt="Command Palette" width="500" height="600"></a></p>
 
+### How to Work with a SuiteCloud Project
+You can do the following actions with a SuiteCloud project:
+1. To create a SutieCloud project, open the Command Palette, enter **SuiteCloud** and, from the dropdown list, select **SuiteCloud: Create Project**.
```
Comment (antoniofruci) [inline packages/vscode-extension/README.md:42]: There's a typo in SuiteCloud

**Review by antoniofruci — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/README.md [hunk 2]
```diff
@@ -30,16 +30,24 @@ See more details about installation in <a href="https://system.netsuite.com/app/
...
+2. To open a SuiteCloud project, go to **File** > **Open Folder...** and select the root folder of your SuiteCloud project.
+
```
Comment (davidecorreu) [inline packages/vscode-extension/README.md:44]: Proposal:

You can do any of the following to start working with a SuiteCloud project:
1. Create a SutieCloud project. Open the Command Palette, enter **SuiteCloud** and, from the dropdown list, select **SuiteCloud: Create Project**.
2. Open a SuiteCloud project. Go to **File** > **Open Folder...** and select the root folder of your SuiteCloud project.

File: packages/vscode-extension/README.md [hunk 3]
```diff
@@ -30,16 +30,24 @@ See more details about installation in <a href="https://system.netsuite.com/app/
...
+### How to Work with Multiple SuiteCloud Projects
+To add a SuiteCloud project to an already opened Visual Studio Code window, go to **File** > **Add Folder To Workspace ...** and select the root folder of your SuiteCloud project. This allows you to work with multiple SuiteCloud projects at the same time.
+
+### Making a File Active in the Editor
+For the extension to detect the working SDF project and run any command against it, you first need to select a file from the current workspace to make it active in the editor.
```
Comment (davidecorreu) [inline packages/vscode-extension/README.md:49]: We should replace SDF project by SuiteCloud project.

File: packages/vscode-extension/README.md (hunk 2 above)
Comment (antoniofruci) [reply packages/vscode-extension/README.md:44]: There's a typo in your proposal @davidecorreu XD

File: packages/vscode-extension/README.md (hunk 3 above)
Comment (Montsant19) [reply packages/vscode-extension/README.md:49]: Yes, this was agreed to be unified to SuiteCloud project but the task has not been started yet. I will change it, in any case.

File: packages/vscode-extension/README.md (hunk 2 above)
Comment (Montsant19) [reply packages/vscode-extension/README.md]: I made a new proposal taking into consideration David's option and fixed the typo.

**Review by davidecorreu — APPROVED** [review]

**Pull request merged**